plugins/dev-scripts/bin/blockly-scripts.js

```diff
@@ -18,7 +18,6 @@ const availableScripts = [
   'start',
   'clean',
   'predeploy',
-  'lint',
```

Comment:
Is this a breaking change for this plugin?

Reply:
Yeah, good call. I'll undo enough changes so that the script still can run lint, it just won't actually be invoked anywhere, and I'll add a deprecation message to the script.